VerifySignature.hs

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env cabal
+
+{- cabal:
+build-depends: base
+             , bytestring
+             , cardano-crypto
+             , cryptonite
+-}
+
+import Crypto.ECC.Ed25519Donna (PublicKey, publicKey, signature, verify)
+import Crypto.Error (throwCryptoErrorIO)
+import qualified Data.ByteString as BS
+
+main :: IO ()
+main = do
+  putStrLn "Loading public key: aggregate.pub"
+  pubKey <- throwCryptoErrorIO . publicKey =<< BS.readFile "aggregate.pub"
+  putStrLn ""
+  verifySignature pubKey "valid.signed"
+  verifySignature pubKey "valid2.signed"
+  verifySignature pubKey "invalid.signed"
```

Comment:
These files aren't committed right? As the `aggregate.pub`. Or, are they artefacts generated by running this `musig2test.c` ?